### ISSUE TITLE
Добавить отображение/выбор плеера Just Player (для IPTV)

### ISSUE THREAD

**Issue by maxkorsov** [lampa-app/LAMPA#21]
Выбор плеера Just Player предлагается при обычном воспроизведении и при при просмотре торрентов, а при просмотре IPTV его нет.

Наблюдаю это при использовании на телефоне и ТВ.

Замечу что в приложении от Nemiroff токой проблемы нет, там Just Player выбирается однажды и используется во всех трех типах воспроизведения уже по умолчанию.

Если возможно это поправить, то уже окончательно перейду на ваше приложение.

**Comment by tsynik** [comment]
> Возможно поправить доступность Just Player при выборе? И объединить выбранный андроид плеер для всех трех типов воспроизведения, в качестве плеера по умолчанию.

Нет. В приложении запрашивается список доступных приложений для каждого типа контента отдельно, чтобы была уверенность, что приложение сможет его открыть. Это нужно исправлять со стороны Just Player объявляя в его манифесте поддержку открытия подобных ссылок.

**Comment by maxkorsov** [comment]
Доверяю вашему ответу, ок. Но странно как тогда в своей версии Lampa TV реализовал это Nemiroff. Там Just Player работает четко, для всех трех воспроизведений.
